src/app/demo/custom-image/custom-image.component.ts

```diff
@@ -0,0 +1,11 @@
+/* tslint:disable:no-access-missing-member */
+import { Component } from '@angular/core';
+import { ImageComponent } from '../../image/image.component';
+
+@Component({
+  selector: 'ru-custom-image',
+  styleUrls: ['custom-image.component.scss'],
+  templateUrl: 'custom-image.component.html'
+})
+export class CustomImageComponent extends ImageComponent {
+}
```

Comment:
i think that you should be able to style the image from the outside. you also do not inherit from the normal `img` tag just to add styles to it, right?